module/book.js

```diff
@@ -0,0 +1,14 @@
+export function createBookElement(book, removeBookCallback) {
```

Comment:
- [ ] In this activity, we really want our code to follow ES6 standards, therefore kindly ensure refactor your functions into arrow function.